docs/neurobagel_environment_variables.tsv

```diff
@@ -1,19 +1,20 @@
-Environment variable	Customization recommended?	Description	Default value if not set	Used in these installation modes
+Environment variable	Default requires update?	Description	Default value if not set	Used in these installation modes
```

Comment:
```suggestion
Environment variable	Default needs change?	Description	Default value if not set	Used in these installation modes
```

Reply:
not sure what's the most clear heading. But "requires update" sounds to me like we need to do something, not the user